### PR TITLE
feat: add command + k shortcut to focus hero search

### LINKED ISSUE #205: Search will have command+K option to show search.
This work should be done in a separate PR from #200. We will need to implement using command + K or Ctrl + K to open the search box. I am sure there is a plugin for keyboard commands, but have not done the research. 

This work cannot merge without #200

new hero

### PR DESCRIPTION
**Pull request by ddsuhaimi**
## What type of PR is this? (check all applicable)

- [X] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
Add cmd/ctrl+k to open search box

## Related Tickets & Documents

fixes #205 
## Mobile & Desktop Screenshots/Recordings
image

https://user-images.githubusercontent.com/30659198/181871277-729441a1-2313-40f8-b521-287037fad7f1.mp4

## Added tests?

- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [X] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
@open-sauced/triage check it out

File: src/components/Hero.tsx [hunk 1]
```diff
@@ -1,17 +1,41 @@
...
 const Hero = () => {
+  const containerRef = useRef<Document>(document);
+  const searchBoxRef = useRef<HTMLInputElement>(null);
   const [searchTerm, setSearchTerm] = useState("");
   const setValueDebounced = useDebounce(setSearchTerm, 500);
   const [fetchedData, setFetchedData] = useState<DbRecomendation[]>([]);
   const [hasFocus, setFocus] = useState(false);
 
+  useKeys(
+    ["ControlLeft", "KeyK"],
```
Comment (bdougie) [inline src/components/Hero.tsx:19]: This works with "Control" but the UI shows "command." I could not find in the docs, but can we include multiple keys? 

The correct key should be "MetaLeft" and not "ControlLeft" - https://www.toptal.com/developers/keycode/for/meta

With that said, I think this should work for windows users.

Comment (ddsuhaimi) [reply src/components/Hero.tsx:19]: I dont think it's possible to use multiple keys, I will have to call two useKeys.

Comment (0-vortex) [reply src/components/Hero.tsx]: We can actually use [useKeyBindings](https://react-hooks.org/docs/useKeyBindings) to bind the 2 pairs to the same callback. I will experiment with this a bit later and check if I can provide some better code, however, feel free to play with the hook, it should work as intended! 🍕

Comment (ddsuhaimi) [reply src/components/Hero.tsx]: Cool, I just updated the PR with possible fix (using two useKeys)

**Review by 0-vortex — APPROVED** [review]
LGTM! 👍

**Comment by 0-vortex** [comment]
Can confirm it works on apple devices, merging this in shortly!

File: src/components/Hero.tsx (hunk 1 above)
Comment (jsmith2320) [reply src/components/Hero.tsx]: It works

**Pull request merged**